README.md

```diff
@@ -1 +1,47 @@
-# CSC750_Project
\ No newline at end of file
+# CSC750_Project
+
+This project has two folders
+
+- Client: Holds the [React](https://reactjs.org/docs/getting-started.html) Project
+- Server: Hols the [Django](https://docs.djangoproject.com/en/4.1/intro/tutorial01/) Project
```

Comment:
Typo on this line, Holds instead of Hols.

Reply:
Resolved